lib/product-import.coffee

```diff
@@ -230,6 +232,52 @@ class ProductImport
       else
         false
 
+  _updateProductRepeater: (prodToProcess, existingProduct) ->
+    repeater = new Repeater {attempts: 5}
```

Comment:
There is this [integration test](https://github.com/sphereio/sphere-product-import/pull/100/files/7e9902a7b01e60537d6d1de654d9c7d393065dde#diff-7ad2ccc25c1229723bd5a6ce06d8ddecR239) which stubs the response of the first update request and returns 409 modification error which should be caught by Repeater.